eth_account/account.py

```diff
@@ -145,6 +145,7 @@ def encrypt(cls, private_key, password, kdf=None):
         :type private_key: hex str, bytes, int or :class:`eth_keys.datatypes.PrivateKey`
         :param str password: The password which you will need to unlock the account in your client
         :param str kdf: The key derivation function to use when encrypting your private key
+        :param str iterations: The work factor for the key derivation function
```

Comment:
I opted for `iterations` instead of `work_factor` to align with the param on the public `create_keyfile_json` function this will call.